deltachat-ffi/deltachat.h

```diff
@@ -3163,6 +3163,54 @@ int             dc_msg_is_setupmessage        (const dc_msg_t* msg);
 char*           dc_msg_get_setupcodebegin     (const dc_msg_t* msg);
 
 
+/**
+ * Check if the message is completely downloaded
+ * or if some further action is needed.
+ *
+ * The function returns one of:
+ * - @ref DC_DOWNLOAD_NOT_NEEDED  - The message does not need any further download action
+ *                                  and should be rendered as usual.
```

Comment:
IMO this should be "DC_DOWNLOAD_NO_URL" which indicates more than "not needed" -- not needed could mean that download was already done but we have covered those cases.  This NO_URL status should just indicate that there is nothing to download at all.

Reply:
k